lib/between_meals/repo/git.rb

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require 'openssl'
```

Comment:
Can you add a comment here that rugged requires this, otherwise we'll nuke it again.

Can you also file a bug with rugged - they should import their own damn requirements.

Actually if you can reference the GH Issue in the comment here, that'd be bonus points.